.github/dependabot.yml

```diff
@@ -62,3 +62,9 @@ updates:
           - "🤖 Dependencies"
       schedule:
           interval: "daily"
+    -   package-ecosystem: "gomod"
```

Comment:
Spacing may be wrong here

Reply:
can you please check if the problem is with `fiberi18n`,  I'm using the spacing like others, and not sure which should be changed?

Reply:
@jcyamacho You are right, problem is the previous item!

Reply:
if you don't mind I can fix it in this pr, just let me know

Reply:
@jcyamacho Sure, thanks!!

Reply:
@gaby done! I'm also added a new config to allow to get a new logger for request, I think it will be useful for integrating opentelemetry tracing id to the logger, or any other custom value.